message_ix_buildings/chilled/util/util.py

```diff
@@ -22,26 +40,27 @@ def get_archs(config: "Config"):
 
             return archs
         else:
-            print(
+            raise TypeError(
```

Comment:
```suggestion
            raise FileNotFoundError(
```
Based on the error message, this and all below should probably be `FileNotFoundError`s, shouldn't they?